### PR TITLE
Remove mui & Updated Tabs

### PR DESCRIPTION
**Pull request by chloeleichen**
### What this PR does
Completely removed MUI dependency
Updated Tabs
screen shot 2018-05-14 at 4 23 07 pm
screen shot 2018-05-14 at 4 26 58 pm

And a few font size adjustments 
### Checklist
- [x] I've updated CHANGES.md with what I changed.
- [x] not applicable

### CODE REVIEW

File: magda-web-client/src/Components/Search/SearchSuggestionBox.js [hunk 1]
```diff
@@ -192,6 +192,7 @@ class SearchSuggestionBox extends Component {
             isMouseOver: false,
             selectedItemIdx: null
         });
+        debugger;
```
Comment (t83714) [inline magda-web-client/src/Components/Search/SearchSuggestionBox.js]: This debug code can be removed.

File: magda-web-client/src/Components/Search/SearchSuggestionBox.js [hunk 2]
```diff
@@ -255,7 +256,7 @@ class SearchSuggestionBox extends Component {
     setupSearchInputListener(newProps) {
         if (!newProps || !newProps.inputRef || !newProps.inputRef.controlEl)
```
Comment (t83714) [inline magda-web-client/src/Components/Search/SearchSuggestionBox.js]:
```
if (!newProps || !newProps.inputRef || !newProps.inputRef.controlEl)
````
can changed to 
```
if (!newProps || !newProps.inputRef)
```

File: magda-web-client/src/Components/SearchFacets/SearchFacets.scss [hunk 3]
```diff
@@ -51,13 +54,10 @@
...
         background-color: #f7f8fa;
+        color: $main;
     }
 }
-
 .search-facets-mobile {
-    margin: 0 -15px;
-    width: calc(100% + 30px);
-
     .facet-header au-btn {
```
Comment (t83714) [inline magda-web-client/src/Components/SearchFacets/SearchFacets.scss]: should `.facet-header au-btn` be `.facet-header .au-btn` (a dot missing before au-btn) ?

Comment (chloeleichen) [reply magda-web-client/src/Components/SearchFacets/SearchFacets.scss]: yeah thanks for spotting that!

**Pull request merged**